### PR TITLE
Fix/235

### PR DESCRIPTION
**Pull request by xross**
Updates to Application Notes.

Requires updated xdoc - please review but don't merge

### CODE REVIEW

**Review by kevinyeungxmos — COMMENTED** [review]
looks good.

**Review by DaveAtkinsonXMOS — COMMENTED** [review]
Probably worth changing the main project readme w.r.t. clock frequency.

File: examples/AN00125_mass_storage_class/Makefile [hunk 1]
```diff
@@ -14,14 +14,14 @@ APP_NAME = app_mass_storage_demo
 
 BUILD_FLAGS = -O3 -report -DUSE_XSCOPE=1 -fxscope -lflash
 
-XCC_FLAGS_xcore200 = $(BUILD_FLAGS) -DXUD_SERIES_SUPPORT=XUD_X200_SERIES
-XCC_FLAGS_xcoreai = $(BUILD_FLAGS)
+XCC_FLAGS_200 = $(BUILD_FLAGS) -DXUD_CORE_CLOCK=500
+XCC_FLAGS_AI = $(BUILD_FLAGS) -DXUD_CORE_CLOCK=600
```
[the remark this answers is not in the document]
Comment (xross) [reply examples/AN00125_mass_storage_class/Makefile:18]: Fixed

File: examples/AN00124_CDC_VCOM_class/Makefile [hunk 2]
```diff
@@ -14,14 +14,14 @@ APP_NAME = app_usb_cdc_demo
 
 BUILD_FLAGS = -O3 -g -report
 
-XCC_FLAGS_xcore200 = $(BUILD_FLAGS) -DXUD_SERIES_SUPPORT=XUD_X200_SERIES
-XCC_FLAGS_xcoreai = $(BUILD_FLAGS)
+XCC_FLAGS_200 = $(BUILD_FLAGS) -DXUD_CORE_CLOCK=500
+XCC_FLAGS_AI = $(BUILD_FLAGS) -DXUD_CORE_CLOCK=600
```
[the remark this answers is not in the document]
Comment (xross) [reply examples/AN00124_CDC_VCOM_class/Makefile:18]: Fixed

File: examples/AN00129_hid_class/doc/rst/AN00129.rst [hunk 3]
```diff
@@ -70,13 +70,7 @@ To start using the USB library, you need to add ``lib_xud`` to your ``Makefile``
...
-You also have to add a define to the build to say which variant of the
-USB library you want. In this case, the example is for the U-Series
-USB library. So the following is in the ``Makefile``::
-
-       XCC_FLAGS = ... -DXUD_SERIES_SUPPORT=XUD_U_SERIES
-
-You can then access the USB functions in your source code via the usb.h header file::
+You can then access the USB functions in your source code via the xud_device.h header file::
 
         #include <usb_device.h>
```
[the remark this answers is not in the document]
Comment (xross) [reply examples/AN00129_hid_class/doc/rst/AN00129.rst]: Fixed

**Comment by xross** [comment]
Merged PR with #169

**Pull request closed without merging**